jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/AsyncTaskTest.java

```diff
@@ -82,7 +82,7 @@ public void testTaskErrorHandling() throws Exception {
     }
 
     @Test(timeout=10000)
-    @BZ("1121027")
+    @Ignore("1121027")
```

Comment:
in case of BZ do we want to use better description like full BZ URL?

Reply:
I would just prepend it with `BZ`, so it would be `BZ-1121027`.

Reply:
it could be enough, even the better to include full URL as you still need to understand what BZ is and how to view it in your browser

Reply:
On the other hand, the URL can change and having to maintain many links can be cumbersome :)

Reply:
Used BZ-123456 format in the new commit. Thank you

Reply:
it could be changed certainly, but usually there is a redirect setup, as it is our jira/bz we can control that by asking to have such redirection in future too ;)